### PR TITLE
[FLINK-12749] Add Flink Operations Playground

### PR DESCRIPTION
**Pull request by fhueske**
## What is the purpose of the change

This PR adds the Flink Operations Playground to the Flink 1.8 documentation.
The playground setup has been pushed to the `flink-playgrounds` repository ([link](https://github.com/apache/flink-playgrounds/tree/release-1.8)).

## Brief change log

* Adds a "Docker Playground" section to the Tutorial section of the documentation
* Adds a description of the Flink Operations Playground

The playground description was backported from the master branch.

## Verifying this change

Check the playground by following its instructions.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

This is a docs only change.

### CODE REVIEW

**Comment by rmetzger** [comment]
@flinkbot approve description consensus

File: docs/tutorials/docker-playgrounds/flink-operations-playground.md [hunk 1]
```diff
@@ -0,0 +1,809 @@
...
+{% endhighlight %}
+
+Afterwards, you can inspect the running Docker containers with the following command:
+
+{% highlight bash %}
+docker-compose ps
+
+                     Name                                    Command               State                   Ports                
+--------------------------------------------------------------------------------------------------------------------------------
+flink-cluster-playground_clickevent-generator_1   /docker-entrypoint.sh java ...   Up       6123/tcp, 8081/tcp
```
Comment (fhueske) [inline docs/tutorials/docker-playgrounds/flink-operations-playground.md]: Update output

**Comment by fhueske** [comment]
PR updated. Please have another look @rmetzger 
Thanks!

**Comment by fhueske** [comment]
Thanks Robert.

Merging.

**Pull request closed without merging**